src/processors/snapshot-processor.ts

```diff
@@ -0,0 +1,10 @@
+// https://eslint.org/docs/developer-guide/working-with-plugins#processors-in-plugins
+
+type PostprocessMessage = { ruleId: string };
```

Comment:
is this not typed from `experimental-utils` somewhere? If not, we should open up an issue with them so it can be added

Reply:
No it's not - I had pegged it for exactly that, but I think the todo got lost in a shuffle 😬

Reply:
Ah that's right - I remember why I was hesitant to open an issue: I couldn't find *any* mention of types for the plugin side of things, so wondered if maybe that wasn't something `@typescript-eslint` was covering.

I'll open an issue up regardless - might as well start somewhere :joy:

Reply:
I've made an [issue for this](https://github.com/typescript-eslint/typescript-eslint/issues/808).